examples/vue/WORKSPACE

```diff
@@ -0,0 +1,22 @@
+workspace(
+    name = "vue_example",
+    managed_directories = {"@npm": ["node_modules"]},
+)
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "build_bazel_rules_nodejs",
+    sha256 = "f9e7b9f42ae202cc2d2ce6d698ccb49a9f7f7ea572a78fd451696d03ef2ee116",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/1.6.0/rules_nodejs-1.6.0.tar.gz"],
```

Comment:
The other examples are at 1.7.0 now, should this be brought up to the same?

Reply:
the release mechanism will auto-update all of these, so it's not really important to have the latest in here at the time we merge